linux/man-section-7.md

```diff
@@ -0,0 +1,61 @@
+# Man Section 7
+
+For most of us, man pages are primarily the source of documentation for various system-level tools, be it `curl`, `ssh`, `gcc` or multitude of others.
```

Comment:
Originally I thought that `man page` is kind of a set phrase on its own, but it probably doesn't read well. Fixed.